hbase_storage/src/main/java/com/splicemachine/access/hbase/H10PartitionAdmin.java

```diff
@@ -422,6 +422,50 @@ public void markDropped(long conglomId, long txn) throws IOException {
         }
     }
 
+    @Override
+    public void enableTableReplication(String tableName) throws IOException {
+        TableName tn = tableInfoFactory.getTableInfo(tableName);
+        try {
+            HTableDescriptor htd = admin.getTableDescriptor(tn);
+            if (admin.isTableEnabled(tn)) {
+                admin.disableTable(tn);
+            }
+
+            for (HColumnDescriptor hcd : htd.getFamilies()) {
+                hcd.setScope(HConstants.REPLICATION_SCOPE_GLOBAL);
+            }
+            admin.modifyTable(tn, htd);
+            SpliceLogUtils.info(LOG, "enabled replication for table %s", tn);
+        }
+        finally {
+            if (tn != null && !admin.isTableEnabled(tn)) {
+                admin.enableTable(tn);
+            }
+        }
+    }
+
+    @Override
+    public void disableTableReplication(String tableName) throws IOException {
+        TableName tn = tableInfoFactory.getTableInfo(tableName);
+        try {
+            HTableDescriptor htd = admin.getTableDescriptor(tn);
+            if (admin.isTableEnabled(tn)) {
+                admin.disableTable(tn);
+            }
+
+            for (HColumnDescriptor hcd : htd.getFamilies()) {
+                hcd.setScope(HConstants.REPLICATION_SCOPE_LOCAL);
+            }
+            admin.modifyTable(tn, htd);
+            SpliceLogUtils.info(LOG, "disabled replication for table %s", tn);
+        }
+        finally {
+            if (tn != null && !admin.isTableEnabled(tn)) {
+                admin.enableTable(tn);
```

Comment:
Similar to the enable replication case, we should probably not enable tables in HBase that started out as disabled before this method was called.